src/app/public/modules/checkbox/checkbox.component.ts

```diff
@@ -152,6 +172,24 @@ export class SkyCheckboxComponent implements ControlValueAccessor {
     this.onTouched();
   }
 
+  public validate(control: AbstractControl): ValidationErrors {
+    if (!this.control) {
+      this.control = control;
```

Comment:
So we're extending the `Validator` class only to pull in the `AbstractControl`?

Reply:
Correct. It was a way to ensure that we skip the initial value. We do the same thing with autocomplete.